azure-spring-cloud-maven-plugin/QUICKSTART.md

```diff
@@ -1,22 +1,33 @@
 # Azure Spring Cloud Maven Plugin Quick Start
 
-Azure Spring Cloud Maven Plugin is intended to help you easily create and update Azure Spring Cloud Services.
- With this tooling, you could run the deploy jobs automatically with pre defined configuration.
+Azure Spring Cloud Maven Plugin helps you easily create and update Azure Spring Cloud Services.
+ This plugin enables clean integration between CI/CD pipelines and the Azure Spring Cloud Service.
 
 > NOTE:
 >
-> More features will be coming after the initial Private Preview announcement.
+> Look out for more features after the initial Private Preview announcement.
 
-### Prerequisite
+### Preconditions
+
+- Azure Subscription, which has been whitelisted to access Azure Managed
+  Service for Spring Cloud.  When you have been added to the whitelist,
+  you will receive an email containing a link to the service landing
+  page.  Save this email!
+
+- JDK 8 or above installed on your local machine
 
-- Azure Subscription, which has been whitelisted to access Azure Managed Service for Spring Cloud
-- JDK 8 installed on your local machine
 - Maven 3.0 and above installed on your local machine
 
+- Use the link from the "Welcome to Azure Spring Cloud" email to
+  provision the Service instance as described in [Provision a service
+  instance on the Azure
+  portal](https://docs.microsoft.com/en-us/azure/spring-cloud/spring-cloud-quickstart-launch-app-maven#provision-a-service-instance-on-the-azure-portal).
 
-### Provision service instance
-### Set Git URL for config server
-Please refer this [document](https://github.com/Azure/azure-managed-service-for-spring-cloud-docs#provision-service-instance) to provision service instance and set config server. 
+- Using `https://github.com/xscript/piggymetrics-config` as the config URI, set the configuration server as described in [Set up your configuration server](https://docs.microsoft.com/en-us/azure/spring-cloud/spring-cloud-quickstart-launch-app-maven#set-up-your-configuration-server).  Return to this document after receiving the "Successfully updated Config Server" message.
+
+- Ensure that the environment that will be running the `mvn` commands is
```

Comment:
This one is not necessary. Even without Azure CLI installed in your environment, the plugin can login with OAuth or Device Login automatically.